eng/versioning/external_dependencies.txt

```diff
@@ -159,9 +159,9 @@ org.assertj:assertj-core;3.16.1
 org.bouncycastle:bcprov-jdk15on;1.60
 org.eclipse.aether:aether-connector-basic;1.1.0
 org.eclipse.aether:aether-transport-wagon;1.1.0
-org.eclipse.jetty:jetty-http;9.4.11.v20180605
-org.eclipse.jetty:jetty-server;9.4.11.v20180605
-org.eclipse.jgit:org.eclipse.jgit;4.5.7.201904151645-r
+org.eclipse.jetty:jetty-http;9.4.30.v20200611
+org.eclipse.jetty:jetty-server;9.4.30.v20200611
+org.eclipse.jgit:org.eclipse.jgit;5.1.3.201810200350-r
```

Comment:
Please don't update the `jgit` dependency. Currently it is only used by resourcemanager, it would cause error when using version after 4.5.

Reply:
Updated. Thank you for your information.